resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -112,3 +112,16 @@ def test_archive_samples(self):
         self.run_process('archive-samples', {
             'data': [txt_file.id, bam.id, reads.id, vcf.id],
             'fields': ['file', 'bam', 'bai', 'fastq', 'fastqc_url', 'fastqc_archive', 'vcf']})
+
+    @tag_process('igv-preprocess-gtf')
+    def test_igv_preprocess(self):
+        with self.preparation_stage():
+            inputs = {'src': 'annotation_non_sorted.gtf',
```

Comment:
It would be nicer if this is rewritten like so::

    inputs = {
        'src': 'annotation_non_sorted.gtf',
        'source': 'ENSEMBL',
        'species': 'Homo sapines',
        'build': 'hg19',
    }
or maybe even::

    annotation = self.run_process('upload-gtf', {
        'src': 'annotation_non_sorted.gtf',
        'source': 'ENSEMBL',
        'species': 'Homo sapines',
        'build': 'hg19',
    })